packages/ref-core/src/cmip_ref_core/pycmec/metric.py

```diff
@@ -0,0 +1,256 @@
+"""
+CMEC metric bundle class
```

Comment:
It would be useful to have a link to the appropriate parts of the spec here

Reply:
Thanks, so, you mean to put the EMDS link within the docstring?

Reply:
Yup